sdk/main/include/Transfer.h

```diff
@@ -102,14 +102,14 @@ class Transfer
   std::shared_ptr<AccountId> mAccountId;
 
   /**
-   * The amount of Hbar transferred.
+   * The amount of Hbar transferred. Defaults to 0.
    */
   Hbar mAmount;
```

Comment:
Missing default = 0 here, but I think it might have been added in a different PR?

Reply:
Default construction of an `Hbar` object sets the amount to zero. I'll add something here to make it more clear.